site/docs/numbers/hostingNumbers.mdx

```diff
@@ -30,34 +30,43 @@ export const Highlight = ({children, color}) => (
     </span>
   );
 
-This walks through how to programmatically import Phone Numbers to your account for use with Bandwidth's Messaging Products.
+In this guide we will show you how to programmatically import Phone Numbers to your account for use with Bandwidth's Messaging Products.
 
+:::info
 Line Options feature management is **NOT** available for Phone numbers that have been imported for use with Hosted Messaging.
+:::
 
-![import_remove_tns](@site/static/img/docs-diagrams/numbers/import-tn-flow.png)
+## Check for Importability
 
-## Polling vs. Webhooks
+[Check TNs portability API](../../../../apis/numbers#tag/accounts/operation/RetrieveOrder) endpoint validates whether or not Bandwidth
+can host the number for messaging.
```

Comment:
I would change lines 41 and 42 to be the following since you already mention the Check TNs API endpoint below.  
```suggestion
In order to determine whether or not Bandwidth can host a number for messaging, you must first check if they can be imported.
```